src/azure-cli/azure/cli/command_modules/vm/_vm_utils.py

```diff
@@ -117,8 +117,8 @@ def get_key_vault_base_url(cli_ctx, vault_name):
 def list_sku_info(cli_ctx, location=None):
     from ._client_factory import _compute_client_factory
 
-    def _match_location(l, locations):
```

Comment:
Is variable name of 1 letter forbidden?

Reply:
May define a variable already as up level.

Reply:
Because it is hard to distinguish between `l` and `1`, `0` and `O`. https://www.flake8rules.com/rules/E741.html